_docs/concepts/network-and-auth/auth.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Auth
+overview: Architectural deep-dive into the design of Auth, which provides the secure communication channel and strong identity for Istio.
+              
+order: 10
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+## Overview
+
+Istio Auth's aim is to enhance the security of microservices and their communication without requiring service code changes. It is responsible for:
+
+
+
+*   Providing each service with a strong identity that represents its role to enable interoperability across clusters and clouds
+
+*   Securing both service to service communication and end-user to service communication
+
+*   Providing a key management system to automate key/cert generation, distribution, rotation, and revocation
+
+In future versions it will also provide:
+
+*   Fine-grained authorization and auditing to control and monitor who accesses your services, apis, or resources
+
+*   Powerful authorization mechanisms: [ABAC](https://en.wikipedia.org/wiki/Attribute-Based_Access_Control), [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control), Authorization hooks
+
+## Architecture
+
+The figure below shows the Istio Auth architecture, which includes three important components: identity, key management, and communication security. It describes how Istio Auth is used to secure the service-to-service communication between service A, running as the service account "foo", and service B, running as the service account "bar".
```

Comment:
the service-to-service -> drop 'the'
as the service account "foo" -> drop 'the'
as the service account "bar" -> the 'the'